ui/src/components/NodePageMetricsTab.js

```diff
@@ -1,8 +1,436 @@
 import React from 'react';
+import { useSelector } from 'react-redux';
+import styled from 'styled-components';
+import {
+  fontSize,
+  padding,
+  fontWeight,
+} from '@scality/core-ui/dist/style/theme';
+import { LineChart } from '@scality/core-ui';
 import { TabContainer } from './CommonLayoutStyle';
+import {
+  addMissingDataPoint,
+  fromUnixTimestampToDate,
+} from '../services/utils';
+import {
+  LAST_SEVEN_DAYS,
+  LAST_TWENTY_FOUR_HOURS,
+  LAST_ONE_HOUR,
+  SAMPLE_DURATION_LAST_SEVEN_DAYS,
+  SAMPLE_DURATION_LAST_TWENTY_FOUR_HOURS,
+  SAMPLE_DURATION_LAST_ONE_HOUR,
+  SAMPLE_FREQUENCY_LAST_SEVEN_DAYS,
+  SAMPLE_FREQUENCY_LAST_TWENTY_FOUR_HOURS,
+  SAMPLE_FREQUENCY_LAST_ONE_HOUR,
+} from '../constants';
+
+const GraphsContainer = styled.div`
+  display: flex;
+  flex-direction: column;
```

Comment:
Just a suggestion: wouldn't it be easier to use CSS Grid instead of a combination of Flexbox containers? Allows for a simpler gap size management, and uniform sizing (looks better when some of the charts are empty, for instance).

Reply:
Actually I didn't see the big difference when come to use CSS Grid. For me, those two can both solve our issue. Maybe just we are more familiar with Flexbox and use it everywhere in our code.

When some of the charts are missing, we'll have an empty chart today. But we should display an error message which is missing today.

Reply:
Won't fight on this topic, just suggesting you try Grid a bit more for building "grid layouts" (which is the case here). There's a reason they built it ;)